server/game/cards/02_SHD/upgrades/HotshotDL44Blaster.ts

```diff
@@ -0,0 +1,38 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { Card } from '../../../core/card/Card';
+import { UpgradeCard } from '../../../core/card/UpgradeCard';
+import { PlayType, Trait } from '../../../core/Constants';
+import Player from '../../../core/Player';
+
+export default class HotshotDL44Blaster extends UpgradeCard {
+    protected override getImplementationId() {
+        return {
+            id: '5874342508',
+            internalName: 'hotshot-dl44-blaster',
+        };
+    }
+
+    public override canAttach(targetCard: Card, controller: Player = this.controller): boolean {
+        if (targetCard.hasSomeTrait(Trait.Vehicle)) {
```

Comment:
I added a one-liner way of doing this since it comes up a lot, take a look at Jedi Lightsaber impl